### ISSUE TITLE
You can skip needing the key for overworld doors using the broom and then dismount in the top part of the door

### ISSUE THREAD

**Issue by Pyrdacor** [Pyrdacor/Ambermoon.net#186]
Reported on discord.

**Comment by Pyrdacor** [comment]
Maybe it is fixable. I am have an idea and will test it.

**Comment by Pyrdacor** [comment]
I managed to fix it! It was a bit tricky but it works now. I tested it with the temple of gala. Now have to fix the rest.

**Comment by Pyrdacor** [comment]
Ok I fixed the temple of gala and crypt doors. That was relatively easy. The only other door affected is the one of Lebab's tower. But this one is tricky because it immediately triggers a fight against some gargoyles.

I also found an issue with that. It seems that if you flee this battle, it won't be happen again. So the gargoyles just vanished then? :D

Maybe I can fix this all together. But it requires some more complex logic and I will only be able to continue this on monday or maybe sunday.
